Disaster Recovery/build-safehaven-base-image-template-from-ubuntu.md

```diff
@@ -0,0 +1,67 @@
+{{{
+  "title": "SafeHaven 3.0/3.1 - Build a base-imaage template from scratch using Ubuntu 14",
+  "date": "9-21-2015",
+  "author": "Jake Malmad",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+#### OVERVIEW:
```

Comment:
@jmalmadclc for consistency with every other article, please make your headings `###` and use upper-lower case, not all caps.